src/renderer/components/dialog-token.tsx

```diff
@@ -120,7 +120,7 @@ export const TokenDialog = observer(
       const text = (clipboard.readText() || '').trim();
 
       if (text.length !== 40) return;
-      if (!/^[a-z0-9]+$/.test(text)) return;
+      if (!/^[a-zA-Z0-9_]+$/.test(text)) return;
```

Comment:
Should we be more specific, modern tokens are `ghp_[a-zA-Z0-9]{n}` (I can't remember what N is)

Reply:
`n` must be 36, because it still is 40 characters long.

If we update it to be more specific, would probably want to also cover the new fine-grained tokens, which look to be in the format `github_pat_[a-zA-Z0-9_]{n}`, with N being 82, at least in a quick spot check, not sure if it changes since they're more complex.

Reply:
The patterns are fixed because they're used for secret scanning

Reply:
Github PAT Regex: https://gist.github.com/magnetikonline/073afe7909ffdd6f10ef06a00bc3bc88